package/common_init/overlay/etc/init.d/copy_duro_eeprom.sh

```diff
@@ -41,6 +41,17 @@ copy_duro_eeprom()
   chmod 0644 "$DURO_EEPROM_TMP_PATH"
   logi "Moving $DURO_EEPROM_TMP_PATH to $DURO_EEPROM_CFG_PATH."
   mv $DURO_EEPROM_TMP_PATH $DURO_EEPROM_CFG_PATH
+
+  # Parse eeprom and set is_duro flag if string "duro" appears in eeprom.
+  # Downstream systems can either read EEPROM contents or check is_duro flag.
+  # pfwp uses is_duro flag as an IPC method to enable can termination setting
+
+  grep -iq duro $DURO_EEPROM_CFG_PATH
```

Comment:
Why relax this constraint?  Currently we're explicitly copying 6 bytes from the eeprom to get the `DUROV0` string exactly.  So, a couple suggestions:
+ keep this as `DUROV0` for now, or maybe shorten to `DUROV` if we want to match future hardware
+ add a constant at the top of the file with the other constants: `DEVICE_DURO_ID_STRING=DUROV`

Reply:
Lets say we change hardware (i.e add wifi to Duro carrier) and  program eeprom to reflect this.  Old firmware won't work correctly on this hardware because is_duro will fail.  Wouldn't it be better to say that DURO mean DURO, and V0, V1, V2, V3 to give us hardware flexibility for future?

Reply:
I am will shorten to DUROV.